docs/fundamentals/jcloud/yaml-spec.md

```diff
@@ -62,7 +62,7 @@ JCloud supports GPU workloads with two different usages: `shared` or `dedicated`
 
 If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G memory for workloads in both usage types.
 
-```{note}
+```{hint}
 When using GPU resources, it may take few extra mins until all Executors ready to serve traffic.
```

Comment:
`When using GPU resources, it may take few extra minutes before all Executors are ready to serve traffic. `